### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gguoss**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,9 +127,9 @@ Check out the new created asset:
 $ ./bytomcli list-assets
 ```
 
-#### Asset issuance test
+#### Issue asset
```
Comment (liuchengxu) [inline README.md]: 'alice' -> `alice`

File: README.md [hunk 2]
```diff
@@ -138,36 +138,31 @@ $ ./bytomcli create-account bob
 Firstly, Alice issue `<issue_amount>`, e.g., 10000, `gold`:
 
 ```bash
-$ ./bytomcli sub-create-issue-tx <alice_account_id> <bob_account_id> <gold_asset_id> <gold_asset_private_key> <issue_amount>
+$ ./bytomcli sub-create-issue-tx <alice_account_id> <gold_asset_id> <issue amount> <gold_asset_private_key> <account_private_key>
```
Comment (liuchengxu) [inline README.md]: issue amount -> issue_amount

File: README.md [hunk 3]
```diff
@@ -138,36 +138,31 @@ $ ./bytomcli create-account bob
...
 
 ```bash
 # Alice should have 10000 gold now
 $ ./bytomcli list-balances
 ```
 
-#### Expenditure test
-
-- Alice -> Bob
+#### Transfer assets(Alice -> Bob)
```
Comment (liuchengxu) [inline README.md]: TOC needs to be updated as well.

File: README.md [hunk 4]
```diff
@@ -138,36 +138,31 @@ $ ./bytomcli create-account bob
...
 
 Alice pays Bob `<payment_amount>`, e.g., 1000, `gold`:
-
+- Bob create receiver program
```
Comment (liuchengxu) [inline README.md]: create -> creates

File: README.md [hunk 5]
```diff
@@ -138,36 +138,31 @@ $ ./bytomcli create-account bob
...
-# In our case, after Alice pays Bob 1000 gold, the amount of Alice's gold should be 9000, Bob's balances should be 1000
-$ ./bytomcli list-balances
+$./bytomcli create-account-receiver bob 
 ```
-
-- Bob -> Alice
-
-Bob pays Alice `<payment_amount>`, e.g., 500, `gold`:
-
+- off-chain transfer receiver program to alice
```
Comment (liuchengxu) [inline README.md]: transfer -> transfers

File: README.md [hunk 6]
```diff
@@ -138,36 +138,31 @@ $ ./bytomcli create-account bob
...
+- Alice build transaction and then make this transacion on-chain
```
Comment (liuchengxu) [inline README.md]: build -> builds, make -> makes

File: README.md [hunk 7]
```diff
@@ -138,36 +138,31 @@ $ ./bytomcli create-account bob
...
 ```bash
-$ ./bytomcli sub-spend-account-tx <bob_account_id> <alice_account_id> <gold_asset_id> <bob_private_key> <payment_amount>
-# In our case, after Bob pays Alice 500 gold, the amount of Alice's gold should be 9500, Bob's balances should be 500
-$ ./bytomcli list-balances
+$./bytomcli sub-control-receiver-tx <account xprv> <account id> <asset id> <spend amount> <control_program>
```
Comment (liuchengxu) [inline README.md]: account xprv -> account_xprv, account id -> account_id, asset id -> asset_id, spend amount -> spend_amount.

File: README.md (hunk 1 above)
Comment (gguoss) [reply README.md]: Accept

File: README.md (hunk 2 above)
Comment (gguoss) [reply README.md]: accept

File: README.md (hunk 4 above)
Comment (gguoss) [reply README.md]: accept

File: README.md (hunk 5 above)
Comment (gguoss) [reply README.md]: accepts

File: README.md (hunk 6 above)
Comment (gguoss) [reply README.md]: Accept

File: README.md (hunk 3 above)
Comment (gguoss) [reply README.md]: accept

**Pull request merged**